mne/io/cnt/cnt.py

```diff
@@ -60,6 +61,12 @@ def read_raw_cnt(input_fname, montage, eog=(), misc=(), ecg=(), emg=(),
         Names of channels or list of indices that should be designated
         EMG channels. If 'auto', the channel names containing 'EMG' are used.
         Defaults to empty tuple.
+    n_bytes : int
+        Defines the number of bytes the data is read in (should be 2 for int16
+        and 4 for int32). Defaults to 2.
```

Comment:
Maybe it would be good to test it against the existing cnt data we have. So maybe, if it's possible, we could read in our testing data in int16 save it as int32 using eeglab and check that the data is about the same. I think the test coverage is still somewhat the same since the only thing changed here is the number of bytes read.